dbscripts/load.sql

```diff
@@ -8,7 +8,9 @@ insert into application_parameters (param_name, param_value) values ('github.tok
 insert into application_parameters (param_name, param_value) values ('application.reporting_email', 'info@example.com');
 insert into application_parameters (param_name, param_value) values ('application.unsubscribe_users', 'false');
 insert into application_parameters (param_name, param_value) values ('application.archive_root', '/tmp/mjolnir-repository-archive');
-insert into application_parameters (param_name, param_value) values ('ldap.url', 'ldap://ldap.example.com');
+insert into application_parameters (param_name, param_value) values ('ldap.url', 'lzz');
```

Comment:
This line change shouldn't be committed.

Reply:
Modified these changes